apps/tangle-dapp/hooks/useExecuteTxWithNotification.ts

```diff
@@ -31,10 +32,12 @@ const useExecuteTxWithNotification = () => {
         }
 
         notificationApi({ variant: 'success', message: successMessage });
-      } catch (error) {
+      } catch (error: any) {
```

Comment:
I think using `unknown` instead of `any`, and then something along the lines of `error instanceof Error ? error.message : 'Something went wrong!'` would be more appropriate here